### ISSUE TITLE
Markdown to markdown conversion losing backslash escape in definition list

### ISSUE THREAD

**Issue by hason** [jgm/pandoc#5543]
Source:

```markdown
1\. item
:   description
```

Command: ``pandoc --from markdown --to markdown``

Bad result:

```markdown
1. item
:   description
```

Expected result:

```markdown
1\. item
:   description
```

https://pandoc.org/try/?text=1%5C.+item%0A%3A+++description&from=markdown&to=markdown

**Comment by mb21** [comment]
What's wrong with that? What's the expected result?

**Comment by hason** [comment]
> What's wrong with that? What's the expected result?

Expected result is 
```
1\. item
:   description
```

**Comment by mb21** [comment]
Ah, of course...

**Comment by jgm** [comment]
We don't preserve information about which characters were escaped in the AST.
But we should check to see if the definition term could be interpreted as a list item, and escape it if so.  We do this already for regular paragraphs, so it's just a matter of adapting that code.

**Comment by jgm** [comment]
Closed by 928681ca049e769bd32c195c6102db5506e7f6fa (typo in commit message referenced wrong issue)
